### PR TITLE
[core] removing dependency on gethue

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- Removing the dependency on gethue

### CODE REVIEW

File: desktop/core/requirements.txt [hunk 1]
```diff
@@ -70,7 +70,7 @@ tablib==0.13.0
 tabulate==0.8.9
 thrift==0.13.0
 thrift-sasl==0.4.2
-git+https://github.com/gethue/django-babel.git
+git+https://github.com/Zegocover/enmerkar.git
```
Comment (Harshg999) [inline desktop/core/requirements.txt:73]: I feel this is as risky as before for depending on external github repo directly. Any change in the master branch of `https://github.com/Zegocover/enmerkar` will affect us.

I think we should move `https://github.com/gethue/django-babel` either under the Cloudera Github organisation (maybe creating the repo their again and referencing here?) or we can also try putting this package in the ext-py3 directory. The latter seems more quicker to me.

Comment (Harshg999) [reply desktop/core/requirements.txt:73]: For python2, we ship the `django-babel` package in ext-py directory: https://github.com/cloudera/hue/tree/master/desktop/core/ext-py/django-babel-0.6.2

Python2 Hue does not depend on `requirements.txt` packages, that is only for Py3.
In ext-py3, we can try putting `https://github.com/gethue/django-babel` package and do the required Makefile build changes similarly how @ranade1 did for adding other packages there!

Comment (agl29) [reply desktop/core/requirements.txt:73]: Yes, I know that.
Please See the changes, need to change installed apps from django_babel to enmerker and same with middleware.

Comment (Harshg999) [reply desktop/core/requirements.txt:73]: But if we are putting the `django-babel` package in ext-py3 itself like other packages there, we don't need to do these changes?

Comment (agl29) [reply desktop/core/requirements.txt:73]: Yes, doing the same thing

**Pull request closed without merging**